packages/vaadin-lumo-styles/components/tabsheet.css

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+@import '../src/mixins/base-layer-reset.css';
+@import '../src/mixins/loader.css';
+@import '../src/components/tabsheet.css';
+@import './scroller.css';
```

Comment:
Note, in tabsheet we use `vaadin-tabsheet-scroller` extension. I wonder if it will get scroller styles, or do we need to also set `--vaadin-tabsheet-scroller-lumo-inject: 1` for it explicitly?

Reply:
Good catch! I've opened a PR to add a visual test for scroller: https://github.com/vaadin/web-components/pull/9611